### ISSUE TITLE
No way to open 'Welcome to UTM' window again while VM is running

### ISSUE THREAD

**Issue by LPFchan** [utmapp/UTM#3632]
**Describe the issue**
<img width="626" alt="Screen Shot 2022-02-16 at 13 25 27" src="https://user-images.githubusercontent.com/1547081/154196283-34597304-ec01-4b11-a948-f8c3795bd4ac.png">
<img width="595" alt="Screen Shot 2022-02-16 at 13 25 40" src="https://user-images.githubusercontent.com/1547081/154196478-869c89bb-f17b-464e-8e1c-d43c6966af98.png">
macOS apps usually assign `⌘+1` to bring up the main window when the window is closed. However UTM lacks that feature so it becomes impossible to open the 'Welcome to UTM' window when it's closed while VM is running on another window.

<img width="390" alt="Screen Shot 2022-02-16 at 13 26 25" src="https://user-images.githubusercontent.com/1547081/154196566-973e388c-4e1f-4333-b34c-5876cf6e7d37.png">
Moreover, I'd expect either `File > New Virtual Machine` or File > Import Virtual Machine... to bring up the main window again, but nothing happens when it's pressed.

**Configuration**
* UTM Version: 3.0.4 and 3.1.0 (beta)
* OS Version: 12.2.1
* Intel or Apple Silicon?: Apple Silicon

**Crash log**
Did not crash

**Debug log**
Not related to VM

**Upload VM**
Not related to VM

**Comment by adespoton** [comment]
The main UTM window can be opened on macOS by clicking the app icon in the dock.  But while that deals with the "no way" part, it's still true that there should be a way from either the File or Window menu to bring the VM selection window back up after it has been closed.

**Comment by osy** [comment]
FB9855975 has been filed with Apple a while ago to address this. SwiftUI does not do this by default and provides no easy mechanism to manually implement it.

**Comment by gronke** [comment]
Currently two behaviors can be observed:

1. Regardless of the last opened VM, the library window always appears on top
2. The library window is closed and can't be re-opened while another VM is running

In the first case the VM library opens, although the intention was to switch back to the last VM. The second case is a bigger issue though, because it prevents from re-opening the VM library at all.

I did not find a way to reproduce one of the two behaviors consistently, but found this thread searching if somebody knows a better workaround than shutting down all VMs to re-access the library.

### Expected behavior
A click on the application dock icon (or opening the application) puts the last open VM or VM library window in focus. The library window can be opened by:
- right click on application icon > "Show Library"
- menu entry: UTM > Show Library
- keyboard shortcut ?

**Comment by Haribo112** [comment]
> The main UTM window can be opened on macOS by clicking the app icon in the dock. But while that deals with the "no way" part, it's still true that there should be a way from either the File or Window menu to bring the VM selection window back up after it has been closed.

This is simply not true. Clicking the UTM icon on my dock will open the console of the running VM. Quitting that window will kill the VM, so I cannot do that. Launching UTM.app from Spotlight also just bring up the running VM console. There does not appear to be any way of bringing back the main UTM / Library window.

**Comment by stewartadam** [comment]
+1, this has been pretty annoying because starting another VM effectively requires you to quit any running ones first so you can restart the app and bring up the main window.

**Comment by osy** [comment]
Yes I agree but it can’t be fixed by us. Apple has to fix it in SwiftUI.

**Comment by gronke** [comment]
A workaround to re-open the library window is to use File > Open > Cancel.

> Yes I agree but it can’t be fixed by us. Apple has to fix it in SwiftUI.

Could you maybe add an extra menu entry, so that it is more clear how to bring back that window?

**Comment by pmthang99** [comment]
Another way is to let UTM show menu bar icon. Then you can see Show UTM option. However, it's a little bit annoying to me when the window is not displayed on top.
